testing/src/test/java/org/motechproject/nms/testing/it/utils/CdrHelper.java

```diff
@@ -384,6 +387,13 @@ private File doMakeCdrFile(String dir, int numInvalidLines) throws IOException {
         return file;
     }
 
+    public static String writeHeader() {
```

Comment:
that doesn't write anything, should be called header() or, even better, make that a string constant.